openedx/adg/admin_dashboard/admin_settings/models.py

```diff
@@ -1,3 +1,53 @@
 """
 Models for `admin_settings` app
 """
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from model_utils.models import TimeStampedModel
+
+from openedx.adg.lms.applications.models import BusinessLine
+
+
+class Invitation(TimeStampedModel):
```

Comment:
Why do we need it to be a time stamped model? According to the designs, the information regarding when an invitation was sent or modified, is not required or mentioned anywhere, as per my knowledge. Please correct me if I'm wrong.